indigo/distribution.yaml

```diff
@@ -17769,6 +17769,16 @@ repositories:
       url: https://github.com/ros/xacro.git
       version: indigo-devel
     status: maintained
+  xaxxon_openlidar:
```

Comment:
@tfoote since Indigo and Lunar are EOL, should the new entries in these two distros be removed from this PR?

Reply:
Yes please only update kinetic and melodic due to the others being EOL.